### ISSUE TITLE
Error building variable fonts (missing from glyph set: zeroinferiorslash.afrc)

### ISSUE THREAD

**Issue by idan** [arrowtype/recursive#363]
**Problem description**
Building the full set on OSX crashes during the variable font mastering build step.

**Expected behavior**
`python build.py -a` results in the fonts being built

**Screenshots**
```
❯ python build.py -a
/Users/idan/src/recursive/mastering/build_static.py:694: SyntaxWarning: "is" with a literal. Did you mean "=="?
  if kind is "otf":
/Users/idan/src/recursive/mastering/build_static.py:702: SyntaxWarning: "is" with a literal. Did you mean "=="?
  elif kind is "ttf":
🚚 Building files for mastering

🚚 Generating sources
🏗  Copying files
🏗  Opening sources
🏗  Checking family name
🏗  Removing non-exporting glyphs
🏗  Clearing guides
🏗  Removing glyphs that aren't in every font
🏗  Removing non-compatible glyphs
🏗  Making kerning compatible
🏗  Sorting glyph order to be common
🏗  Setting production names
🏗  Setting version
🏗  Closing and saving sources
🏗  Checking full design space
🏗  Writing report
✅ Done preparing sources
🏗  Moved features into UFOs

🚚 Making files for static font mastering
🏗  Made folders for static fonts
🏗  Made FontMenuDB
🏗  Made Family features
🏗  Generating instance UFOs
🏗  Getting blueScale
🏗  Setting values, removing overlap, writing files
   |██████████████████████████████████████████████████| 100.0% Complete
✅ Made UFO instances
🏗  Running checkoutlinesUFO on files
   |██████████████████████████████████████████████████| 100.0% Complete

🚚 Making files for varible font mastering
🏗  Made stylespace
🏗  Constructing variable font
Traceback (most recent call last):
  File "build.py", line 64, in <module>
    build_variable(designspacePath=files["designspace"],
  File "/Users/idan/src/recursive/mastering/build_variable.py", line 156, in build_variable
    fp.build_variable_font(designspacePath,
  File "/Users/idan/src/recursive/venv/lib/python3.8/site-packages/fontmake/font_project.py", line 298, in build_variable_font
    font = ufo2ft.compileVariableTTF(
  File "/Users/idan/src/recursive/venv/lib/python3.8/site-packages/ufo2ft/__init__.py", line 590, in compileVariableTTF
    ttfDesignSpace = compileInterpolatableTTFsFromDS(
  File "/Users/idan/src/recursive/venv/lib/python3.8/site-packages/ufo2ft/__init__.py", line 422, in compileInterpolatableTTFsFromDS
    for source, ttf in zip(result.sources, ttfs):
  File "/Users/idan/src/recursive/venv/lib/python3.8/site-packages/ufo2ft/__init__.py", line 320, in compileInterpolatableTTFs
    compileFeatures(
  File "/Users/idan/src/recursive/venv/lib/python3.8/site-packages/ufo2ft/__init__.py", line 550, in compileFeatures
    otFont = featureCompiler.compile()
  File "/Users/idan/src/recursive/venv/lib/python3.8/site-packages/ufo2ft/featureCompiler.py", line 125, in compile
    self.setupFeatures()
  File "/Users/idan/src/recursive/venv/lib/python3.8/site-packages/ufo2ft/featureCompiler.py", line 220, in setupFeatures
    featureFile = parseLayoutFeatures(self.ufo)
  File "/Users/idan/src/recursive/venv/lib/python3.8/site-packages/ufo2ft/featureCompiler.py", line 48, in parseLayoutFeatures
    doc = parser.parse()
  File "/Users/idan/src/recursive/venv/lib/python3.8/site-packages/fontTools/feaLib/parser.py", line 88, in parse
    statements.append(self.parse_glyphclass_definition_())
  File "/Users/idan/src/recursive/venv/lib/python3.8/site-packages/fontTools/feaLib/parser.py", line 254, in parse_glyphclass_definition_
    glyphs = self.parse_glyphclass_(accept_glyphname=False)
  File "/Users/idan/src/recursive/venv/lib/python3.8/site-packages/fontTools/feaLib/parser.py", line 349, in parse_glyphclass_
    self.check_glyph_name_in_glyph_set(glyph)
  File "/Users/idan/src/recursive/venv/lib/python3.8/site-packages/fontTools/feaLib/parser.py", line 1602, in check_glyph_name_in_glyph_set
    raise FeatureLibError(
fontTools.feaLib.error.FeatureLibError: /Users/idan/src/recursive/mastering/build/src/./features/zero.fea:3:81: The following glyph names are referenced but are missing from the glyph set: zeroinferiorslash.afrc

recursive/mastering on  main [$] via 🐍 v3.8.5 (venv) took 1h3m17s 
```

**To Reproduce**
1. `python build.py -a`
2. 💥 (eventually)

**Environment (please complete the following information):**
- OS: MacOS 10.15.6 (Catalina), python 3.8.5 from homebrew
- Fonts: building from master

**Additional context**
My true goal is to build tweaked versions of the code variants, but it seems like I have to first build the variable fonts and then use an edited `instantiate_code_fonts.py` to generate the fixed code representations.

Thank you for recursive! ❤️

**Comment by arrowtype** [comment]
Thanks! This should probably be fixed once #359 is completed.

**Comment by benkiel** [comment]
Fixed with https://github.com/arrowtype/recursive/pull/379
